Frameworks/Foundation/NSCache.mm

```diff
@@ -286,7 +286,7 @@ - (void)_evictEntriesOverCharge {
 
     auto rit = _cacheEntries.end();
     do {
-        _NSCacheEntry* entry = (id)*--rit;
+        _NSCacheEntry* entry = (id) * --rit;
```

Comment:
I don't feel too strongly about it, but I'd rather not pull in any formatting only changes.

---
In reply to: [72163413](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72163413) [](ancestors = 72163413,72162462,70353130,70347524)